### DISCUSSION TITLE
Opening a single file on Linux

### DISCUSSION THREAD

**Discussion by SyntheticBird45** [zed-industries/zed#12538] [Feature Requests]
Is there no way to open a single file in Zed ? `workspace: open` only let me select folders.

**Comment by versecafe** [comment]
Does File >  Open ... or ctrl + O not work?

**Reply by SyntheticBird45** [reply]
No. Files are greyed out and I can't open them

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
